feat(changelog): messaging-and-queuing-changed-messaging-and-queuing-is- 2023-10-24 (#2307)
scaleway/docs-content: changelog/october2023/2023-10-24-messaging-and-queuing-changed-messaging-and-queuing-is-.mdx

---
title: Messaging and Queuing is now in General Availability
status: changed
author:
    fullname: 'Join the #messaging-queuing channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-10-24
category: serverless
product: messaging-and-queuing
---

> Messaging and Queuing (M&Q) proposes a message broker solution, so you can send messages & trigger events between your Cloud resources.

Reviewer: ```suggestion
Messaging and Queuing (M&Q) proposes a message broker solution, so you can send messages & trigger events between your cloud resources.
```

> Messaging and Queuing is Scaleway's message broker solution that allows you to send messages & trigger events between your cloud resources.

Reviewer 2: ```suggestion
Messaging and Queuing is Scaleway's message broker solution that allows you to send messages and trigger events between your cloud resources.
```

> The service supports 3 protocols (NATS, SQS & SNS still in beta) for you to work with. Messaging and Queuing is serverless-based, so you can run it without worrying about provisioning and you only pay for what you use.

Reviewer 2: ```suggestion
The service supports 3 protocols: NATS, SQS and SNS (the latter is still in Public Beta). Messaging and Queuing is serverless-based, so you can run it without worrying about provisioning and you only pay for what you use.
```

> M&Q features are available though Console, CLI and Terraform.

Reviewer: ```suggestion
M&Q features are available though console, CLI and Terraform.
```

> M&Q features are available though Console, CLI and Terraform.

Reviewer 3: ```suggestion
The Messaging and Queuing features are available via the Scaleway Console, the CLI and Terraform.
```

> Your existing resources will need to be migrated, as communicated to all beta users. 

Reviewer 2: ```suggestion
Your existing resources will need to be [migrated](/serverless/messaging/reference-content/migration/), as communicated to all beta users. 
```

> Console & API have changed, so existing resources will need to be migrated as communicated to all beta users. Documentation and the team are there to give you the best support.

Reviewer 3: ```suggestion
Your existing resources will need to be migrated, as communicated to all beta users. 
```

> Find out more in our dedicated M&Q documentation.

Reviewer 3: ```suggestion
Find out about Messaging and Queuing in our [dedicated documentation](/serverless/messaging/quickstart/). If you have any questions, reach out to the team on the #messaging-queuing community Slack channel.
```
